src/mappings/core.ts

```diff
@@ -473,75 +464,5 @@ export function handleSwap(event: SwapEvent): void {
     pool.save()
     token0.save()
     token1.save()
-
-    // Update inner vars of current or crossed ticks
-    let newTick = pool.tick
-    if (oldTick && newTick) {
-      let tickSpacing = feeTierToTickSpacing(pool.feeTier)
-      let modulo = newTick.mod(tickSpacing)
-      if (modulo.equals(ZERO_BI)) {
-        // Current tick is initialized and needs to be updated
-        loadTickUpdateFeeVarsAndSave(newTick.toI32(), event)
-      }
-
-      let numIters = oldTick
-        .minus(newTick)
-        .abs()
-        .div(tickSpacing)
-
-      if (numIters.gt(BigInt.fromI32(100))) {
-        // In case more than 100 ticks need to be updated ignore the update in
-        // order to avoid timeouts. From testing this behavior occurs only upon
-        // pool initialization. This should not be a big issue as the ticks get
-        // updated later. For early users this error also disappears when calling
-        // collect
-      } else if (newTick.gt(oldTick)) {
-        let firstInitialized = oldTick.plus(tickSpacing.minus(modulo))
-        for (let i = firstInitialized; i.le(newTick); i = i.plus(tickSpacing)) {
-          loadTickUpdateFeeVarsAndSave(i.toI32(), event)
-        }
-      } else if (newTick.lt(oldTick)) {
-        let firstInitialized = oldTick.minus(modulo)
-        for (let i = firstInitialized; i.ge(newTick); i = i.minus(tickSpacing)) {
-          loadTickUpdateFeeVarsAndSave(i.toI32(), event)
-        }
-      }
-    }
-  }
-}
-
-export function handleFlash(event: FlashEvent): void {
```

Comment:
maybe not for this PR shouldnt we be checking state diff in flash events as well? for general things like TVLs change for the tokens involved? 

potentially not if flash gaurantees the that balances remain the same kinda forget how it works

Reply:
noted!